SwiftyBluetooth/Source/Central.swift

```diff
@@ -62,6 +62,18 @@ public enum PeripheralScanResult {
 }
 
 /**
+ The different results returned in the closure of the Central connect(...) and disconnect(...) functions.
```

Comment:
This makes sense 😄 the Result type structure in the future sounds good, will make sure to merge that upstream change when you add it.